src/main/java/frc/robot/Robot.java

```diff
@@ -77,6 +98,37 @@ public void robotPeriodic() {
     LogManager.log();
   }
 
+    /**
+   * Method to store DPad values and use them to set selectedNode
+   * Down clears the array
+   * Left is 1, up is 2, and right is 3 for selection
+   * For example, up right left will select the center grid, top row, and left spot.
+   * @param direction = Which DPad button is pressed
+   */
+  public static void DPadPress(DPad direction) { 
```

Comment:
selectTime and selectValues are not very clear, additionally, this should be in Driver.java or Operator.java